.github/update-sync-param-files.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import glob
+
+import yaml
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("output", type=str, help="output path of sync-param-files.yaml")
+    args = parser.parse_args()
+
+    # load current sync-param-files.yaml
+    with open(args.output) as f:
+        sync_param_file_curr = yaml.safe_load(f)
+    src2dst_curr = {}
+    for src_and_dst in sync_param_file_curr[0]["files"]:
+        src, dst = src_and_dst["source"], src_and_dst["dest"]
+        if src not in src2dst_curr.keys():
+            src2dst_curr[src] = []
+        src2dst_curr[src].append(dst)
+
+    # create a list of .param.yaml files within autoware.universe/launch directory
+    autoware_universe_path = "/tmp/autoware.universe/"
```

Comment:
Please do not hardcode the path here.

Reply:
I changed the python script in this commit https://github.com/autowarefoundation/autoware_launch/pull/75/commits/91e4658a442060a092ee17c1ace677bec7603851
(same goes for the rest of your comments)

Reply:
Looks clean! :smile:

Reply:
Great to hear that, thanks!